Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroupBehavior.cpp

```diff
@@ -51,19 +81,290 @@ namespace AZ::SceneAPI::Behaviors
+        Containers::Scene& scene,
+        ManifestAction action,
+        [[maybe_unused]] RequestingApplication requester)
+    {
+        if (action != Events::AssetImportRequest::ConstructDefault)
+        {
+            return Events::ProcessingResult::Ignored;
+        }
+
+        // this toggle makes constructing default mesh groups and a prefab optional
+        if (AZ::SettingsRegistryInterface* settingsRegistry = AZ::SettingsRegistry::Get(); settingsRegistry)
+        {
+            bool createDefaultPrefab = true;
+            settingsRegistry->Get(createDefaultPrefab, s_PrefabGroupBehaviorCreateDefaultKey);
+            if (createDefaultPrefab == false)
+            {
+                return Events::ProcessingResult::Ignored;
+            }
+        }
+
+        using MeshTransformPair = AZStd::pair<Containers::SceneGraph::NodeIndex, Containers::SceneGraph::NodeIndex>;
+        using MeshTransformEntry = AZStd::pair<Containers::SceneGraph::NodeIndex, MeshTransformPair>;
+        using MeshTransformMap = AZStd::unordered_map<Containers::SceneGraph::NodeIndex, MeshTransformPair>;
+        MeshTransformMap meshTransformMap;
+
+        auto graph = scene.GetGraph();
+        const auto view = Containers::Views::MakeSceneGraphDownwardsView<Containers::Views::BreadthFirst>(
+            graph,
+            graph.GetRoot(),
+            graph.GetContentStorage().cbegin(),
+            true);
+
+        if (view.begin() == view.end())
+        {
+            return Events::ProcessingResult::Ignored;
+        }
+        for (auto it = view.begin(); it != view.end(); ++it)
+        {
+            Containers::SceneGraph::NodeIndex currentIndex = graph.ConvertToNodeIndex(it.GetHierarchyIterator());
+            AZStd::string currentNodeName = graph.GetNodeName(currentIndex).GetPath();
+            auto currentContent = graph.GetNodeContent(currentIndex);
+            if (currentContent)
+            {
+                if (azrtti_istypeof<AZ::SceneAPI::DataTypes::ITransform>(currentContent.get()))
+                {
+                    auto parentIndex = graph.GetNodeParent(currentIndex);
+                    if (parentIndex.IsValid() == false)
+                    {
+                        continue;
+                    }
+                    auto parentContent = graph.GetNodeContent(parentIndex);
+                    if (parentContent && azrtti_istypeof<AZ::SceneAPI::DataTypes::IMeshData>(parentContent.get()))
+                    {
+                        // map the node parent to the ITransform
+                        MeshTransformPair pair {parentIndex, currentIndex};
+                        meshTransformMap.emplace(MeshTransformEntry{ graph.GetNodeParent(parentIndex), AZStd::move(pair) });
+                    }
+                }
+            }
+        }
+
+        if (meshTransformMap.empty())
+        {
+            return Events::ProcessingResult::Ignored;
```

Comment:
I wonder if a scene with no mesh data nodes is just a valid FBX file. For example, maybe the team broken out the animations away from the mesh scenes. It might be concerning if that team saw a lot of warnings then.